src/components/Person.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './Person.scss';
+
+import PropTypes from 'prop-types';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
+    </h2>
+    {age ? (
+      <p className="Person__age">
+        { `I am ${age}`}
+      </p>
+    ) : null}
+
+    {isMarried ? (
+      <p className="Person__partner">
+        {sex === 'm'
+          ? `My wife's name is ${partnerName}`
+          : `My husband's name is ${partnerName}`
+        }
```

Comment:
```suggestion
        {`My ${sex === 'm' ? 'wife' : 'husband'}'s name is ${partnerName}`}
```
Можна трішки скоротити